Return forum URL with the SSO login params

wil93 opened the pull request:
This makes it easier for the frontend to redirect to the correct URL

veluca93 reviewed (commented):
Why not leave parameters as it is, and add a path property to the returned value?

wil93 commented:
No specific reason, we can also just keep them separate and build it into a URL from the frontend.

I updated it, and also changed the way we parse the two parameters (using `parse_qs`)
